From 3bc24e232913684830edb2e029d1695f3470308e Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Tue, 8 Mar 2016 14:52:37 -0800
Subject: [PATCH] Ticket #48757 - License tag does not match actual license of
 code

Fix Description: License tag in the spec file was updated to match the
actual license of code.

(cherry picked from commit 415d8553eddfe9b19fd9b8d8be9b934bf75a5765)
(cherry picked from commit c9bd68813c658e1caeaba9ee2c2aef62f0cc6dd6)
---
 rpm/389-ds-base.spec.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/389-ds-base.spec.in b/rpm/389-ds-base.spec.in
index b7556e1..8bbd979 100644
--- a/rpm/389-ds-base.spec.in
+++ b/rpm/389-ds-base.spec.in
@@ -33,7 +33,7 @@ Name:             389-ds-base
 Version:          __VERSION__
 #Release:          %{?relprefix}1%{?prerel}%{?dist}
 Release:          __RELEASE__%{?dist}
-License:          GPLv2 with exceptions
+License:          GPLv3+
 URL:              http://port389.org/
 Group:            System Environment/Daemons
 BuildRoot:        %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
-- 
2.4.3

